### PR TITLE
^^^MAP FREEZE ABOVE THIS POINT!!!^^^

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Simply put, with the huge revamp to pipenets, all map related changes should be delayed until that is merged in, as that PR literally conflicts with even tiny tiny map changes.

### CODE REVIEW

**Comment by tigercat2000** [comment]
Enough labels?

File: _maps/cyberiad.dm [hunk 1]
```diff
@@ -1,34 +1 @@
...
-        #define MAP_FILE "cyberiad.dmm"
-        #define MAP_NAME "NSS Cyberiad"
-
-#elif !defined(MAP_OVERRIDE)
-
-	#warn a map has already been included, ignoring Cyberiad.
-
-#endif
\ No newline at end of file
+SO I HURD YOU LIKE RUNTIMES //obviously compiles. No, don't question it.
```
Comment (tigercat2000) [inline _maps/cyberiad.dm]: at least it isn't an HTTP:// link at the top of a file

**Comment by Keilden** [comment]
Maintainer Label Abuse      Sounds about right

**Comment by SirGaroon** [comment]
Not Enough Labels™ brand label is required

**Comment by Alex6511** [comment]
Cannot understand why no compile.

**Comment by tigercat2000** [comment]
@Alex6511 obviously travis questioned it

**Comment by Alex6511** [comment]
TRAVIS IS WRONG THIS COMPILE YAYA

**Comment by Markolie** [comment]
What abuse?

**Pull request closed without merging**